.travis.yml

```diff
@@ -78,7 +78,7 @@ matrix:
       name: "Python nightly with Extensions"
     - env: TOX_ENV=pyNightly-no-ext
       python: 'nightly'
-      name: "Python nightly Extensions"
+      name: "Python nightly without Extensions"
```

Comment:
Need to also make the change to the name up on line 74 so it does not break the build. This fails Travis on this build, but it should allow failure.

Reply:
@ahopkins Fix implemented, can you finish the review?